### DISCUSSION TITLE
[produce] Select Prefix ID

### DISCUSSION THREAD

**Discussion by MartinGonzalez** [fastlane/fastlane#16999] [Q&A]
Hi there!

It's a mix of a question/feature request. Can I choose the prefix id when using produce?
If not, is it possible?

Thanks!

**Comment by MartinGonzalez** [comment]
Any idea about this?
Using produce uses TeamId as default prefix.

**Comment by MartinGonzalez** [comment]
:(

**Comment by Thompsons21** [comment]
First thoughts, which seems to work well:

$('div[id^="my"]').filter(
    function(){
        return this.id.match(/\d+$/);
    });
[JS Fiddle demo](http://jsfiddle.net/davidThomas/9QL8y/).

The above selects all div elements whose id starts with the value my, and then filters the returned elements to those whose id also ends with numeric characters.

References:

[attribute-starts-with selector](http://api.jquery.com/attribute-starts-with-selector/).
[filter()](http://api.jquery.com/filter/).
[Regular Expressions, at Mozilla Developer Network](https://developer.mozilla.org/en/JavaScript/Guide/Regular_Expressions). 
[KohlsFeedback Survey Code](https://www.kohlsfeedback.org/)
